content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -64,31 +61,76 @@ To combine an event, click the three dots next to an event and select **+ Combin
+
+
+<div class="alert alert-info"> 
+
+If you select <code>Session</code>, all steps must be completed within the same <code>@session.id</code> to count as a conversion.
+
+If you select <code>User</code>, the funnel requires the same individual user (<code>@user.id</code>) to complete every step for the conversion to count.
+
+If you select <code>Account</code>, different users within the same account can complete different steps and the conversion still counts. In this case, the funnel is tied to the <code>@account.id</code> facet which maps to org IDs in Datadog.  
+
+</div>
 
 You can measure conversion by the following attributes:
 
-- **Conversion count** - A count of users who went through the funnel you've defined.
-- **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+- **Conversion count**: A count of users who went through the funnel you've defined.
+- **Conversion rate**: A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
 
   **Note**: Only completed RUM sessions contribute to conversion rates.
 
-- **Time to convert** - The time it took for the user to complete the step events.
+- **Time to convert**: The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the **filter** selector to filter by various criteria that you define. These filters are applied to all steps of the funnel.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## How Datadog computes conversion metrics
+
+Consider a funnel with events `A → B → C` and event steps: 
+
+```
+A, A, A, B, C, C
+```
+
+If a user performs the actions as in the example above, Datadog counts it as one conversion. This is because the conversion calculations only look at the first element **A** matched and the first element **C** matched. 
+
+<div class="alert alert-info">
+If the user performs the event steps <code>A, A, A, B, C, C, A,...</code>, it would count as two conversions. The first conversion ends at <code>A, A, A, B, C, C</code>, and the second conversion begins with <code>A,...</code>
+
+</div>
+
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnel_analysis_conversion.png" alt="Select a conversion measure, whether Unique or Total, to determine how your session conversions are counted." style="width:80%;" >}}
```

Comment:
This screenshot doesn't have any context associated with it, so I wasn't initially sure what to do with this information. There was a pretty sharp transition between conceptual and procedural content - maybe add in some steps to smooth that transition so users know when they're supposed to be learning vs. when they're supposed to be doing something in the UI?